src/QMCWaveFunctions/tests/lcao_spinor_test.py

```diff
@@ -130,13 +130,13 @@ def val(self,pos):
         norm = self.norm()
```

Comment:
I saw above an h5 file was removed. No more needed?

Reply:
the file wasn't removed, it just got overwritten. The lcao_spinor_test.py writes the h5 file that is read by SPOSetBuilder in the test_MO_spinor.py. When I reran it to add new reference values, it just overwrote  that file